packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2744,8 +2744,6 @@ class InputDecoration {
   ///
   /// When [FloatingLabelBehavior.never] the label will always appear in an empty
   /// field in place of the content.
-  ///
-  /// Defaults to [FloatingLabelBehavior.auto].
```

Comment:
Could you add this line to the InputDecorationTheme where this template is used?  Since there, it does default to auto.

Also here, outside of the template, maybe specify that the theme will be used:

"If null, [InputDecorationTheme.floatingLabelBehavior] will be used."